lib/resources/dstu2/condition.yaml

```diff
@@ -139,12 +139,12 @@ fields:
     }
   binding:
     description: A category assigned to the condition.
-    note: Currently `diagnosis` and `problem` category codes are supported. Code `diagnosis` is bound to condition category codes and `problem` is bound to argonaut extenstion codes terminology system shown below.
+    note: Category codes `diagnosis`, `problem` and `health-concern` are currently supported for search and retrieve functionality. Code `health-concern` is currently not supported for conditions add and update functionality.
```

Comment:
I don't think the definition of what action supports which category belongs here. That should be in the implementation notes of each action.